app/core/resourceRegistryClient_test.go

```diff
@@ -0,0 +1,98 @@
+package core
+
+import (
+	"github.com/cloudogu/k8s-ces-setup/app/context"
+	"github.com/cloudogu/k8s-ces-setup/app/core/mocks"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestNewResourceRegistryClient(t *testing.T) {
+	t.Run("successful init", func(t *testing.T) {
+		// given
+		doguRegistrySecret := &context.DoguRegistrySecret{
+			Endpoint: "endpoint",
+			Username: "username",
+			Password: "password",
+		}
+
+		// when
+		sut := NewResourceRegistryClient("1.0.0", doguRegistrySecret)
+
+		// then
+		require.NotNil(t, sut)
+		assert.Equal(t, doguRegistrySecret, sut.doguRegistrySecret)
+		assert.NotNil(t, sut.fileClient)
+	})
```

Comment:
You do not require a wrapper if it is the only test

```suggestion
	 // given
	doguRegistrySecret := &context.DoguRegistrySecret{
		Endpoint: "endpoint",
		Username: "username",
		Password: "password",
	}
	
	// when
	sut := NewResourceRegistryClient("1.0.0", doguRegistrySecret)

	// then
	require.NotNil(t, sut)
	assert.Equal(t, doguRegistrySecret, sut.doguRegistrySecret)
	assert.NotNil(t, sut.fileClient)
```